src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceEventSource.cs

```diff
@@ -355,8 +355,8 @@ protected override void OnEventCommand(EventCommandEventArgs command)
                 if ((command.Command == EventCommand.Update || command.Command == EventCommand.Enable) &&
                     IsEnabled(EventLevel.Informational, Keywords.Events))
                 {
-                    string filterAndPayloadSpecs;
-                    command.Arguments.TryGetValue("FilterAndPayloadSpecs", out filterAndPayloadSpecs);
+                    string? filterAndPayloadSpecs = null;
+                    command.Arguments?.TryGetValue("FilterAndPayloadSpecs", out filterAndPayloadSpecs);
```

Comment:
This is IL change, but I think its better to use it instead !

Reply:
> but I think its better to use it instead !

Why?  Is there a path here where command.Arguments would be null such that this would otherwise null ref?

Reply:
Because command/event arguments can be null, and we make them all nullable within this project too. But as it is internal class might not necessarily check it, will update with !.